looprpc/client.proto

```diff
@@ -62,6 +64,12 @@ service SwapClient {
     */
     rpc GetLoopInQuote (QuoteRequest) returns (InQuoteResponse);
 
+    /* loop: `probe`
```

Comment:
The subcommand for this is `quote in`. (This is used by the API doc generator to extract the CLI help text).